README.md

```diff
@@ -326,17 +328,17 @@ for the value to be credited to your account.
 
 ## Issuing
 
-### Query Issuing BINs
+### Query Issuing Products
 
-To take a look at the sub-issuer BINs linked to your workspace, just run the following:
+To take a look at the sub-issuer products linked to your workspace, just run the following:
```

Comment:
To take a look at the sub-issuer card products available to you, just run the following: